x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/enrich/AbstractLookupService.java

```diff
@@ -127,14 +126,25 @@
  * </p>
  */
 public abstract class AbstractLookupService<R extends AbstractLookupService.Request, T extends AbstractLookupService.TransportRequest> {
+
+    /**
+     * Field for Positions in lookup operations. Contains an IntBlock of positions.
```

Comment:
Would this be more precise?
```suggestion
     * Represents an integer column that corresponds to the positions of the page that we perform the lookup for.
```

Reply:
Done